### PR TITLE
Playlist Support + mm:ss.xx lyrics time

### LINKED ISSUE #311: [Feature Request] Add an option to create a m3u playlist file for open files
I'm currently using Kid3 as my music tagger of choice but I want to use Tagger as I like its UI much more. But there is one feature missing that prevents me from using it as my default tagger and that is the creation of a playlists.
I find this to be very handy while modifying the tags and names of my audio files to reflect the changes in my playlists to keep them functional.
There is also no program currently in the current gnome core apps or gnome circle ecosystem to provide playlist creation and handling.
I think this would be a great fit for this application.

### LINKED ISSUE #321: Unable to import synchronized lyrics
I use Tagger from Flatpak, version 2023.9.0, on Fedora 38.

I try to use a lrc file to add synchronized lyrics to a song. The lyrics I have are pretty simple and follow the lrc standard:
[mm:ss.xx]Lyrics

They are not imported by Tagger and I don't get any error message either. I can share the lyrics file if that helps.

It's also seems strange to me that manually adding synchronized lyrics asks for the format [hh:mm:ss]:

image

While lrc files use the [mm:ss.xx] format (notice the . between ss and xx.) Maybe this has just something to do with what is displayed and is not linked in any way to the issue I am facing now.

### PR DESCRIPTION
**Pull request by nlogozzo**
Fixes #311 
Fixes #321

### CODE REVIEW

File: NickvisionTagger.GNOME/Views/LyricsDialog.cs [hunk 1]
```diff
@@ -238,18 +238,18 @@ private bool OnClose(Gtk.Widget sender, EventArgs e)
     /// <param name="e">EventArgs</param>
     private void AddSyncLyric(Gtk.Button sender, EventArgs e)
     {
-        var entryDialog = new EntryDialog(this, _iconName, _("New Synchronized Lyric"), "", _("Timestamp (hh:mm:ss)"), _("Cancel"), _("Add"))
+        var entryDialog = new EntryDialog(this, _iconName, _("New Synchronized Lyric"), "", _("Timestamp (hh:mm:ss, mm:ss.xx)"), _("Cancel"), _("Add"))
```
Comment (fsobolev) [inline NickvisionTagger.GNOME/Views/LyricsDialog.cs:241]:
```suggestion
        var entryDialog = new EntryDialog(this, _iconName, _("New Synchronized Lyric"), "", _("Timestamp (hh:mm:ss or mm:ss.xx)"), _("Cancel"), _("Add"))
```

**Review by fsobolev — CHANGES_REQUESTED** [review]

File: NickvisionTagger.GNOME/Views/LyricsDialog.cs (hunk 1 above)
Comment (fsobolev) [reply NickvisionTagger.GNOME/Views/LyricsDialog.cs]: I understand but I guess some people may be unfamilliar with writing timestamp and will think that `hh:mm:ss, mm:ss.xx` is somehow one format, not 2 examples.

**Comment by nlogozzo** [comment]
Moving back to draft as we are going to add full support for playlists into Tagger meaning editing and managing it's files as if you were to open it as a folder

File: NickvisionTagger.GNOME/Views/LyricsDialog.cs (hunk 1 above)
Comment (nlogozzo) [reply NickvisionTagger.GNOME/Views/LyricsDialog.cs]: > I understand but I guess some people may be unfamilliar with writing timestamp and will think that `hh:mm:ss, mm:ss.xx` is somehow one format, not 2 examples.

Yeah true

File: NickvisionTagger.GNOME/Controls/CreatePlaylistDialog.cs [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    {
+        
+    }
+
+    /// <summary>
+    /// Validates the dialog's input
+    /// </summary>
+    private void Validate()
+    {
+        var valid = !string.IsNullOrEmpty(_nameRow.GetText());
```
Comment (fsobolev) [inline NickvisionTagger.GNOME/Controls/CreatePlaylistDialog.cs:74]: We should alsocheck for invalid filename/path characters

**Review by fsobolev — CHANGES_REQUESTED** [review]

File: NickvisionTagger.GNOME/Controls/CreatePlaylistDialog.cs (hunk 2 above)
Comment (nlogozzo) [reply NickvisionTagger.GNOME/Controls/CreatePlaylistDialog.cs:74]: Would you like to implement that?

Comment (fsobolev) [reply NickvisionTagger.GNOME/Controls/CreatePlaylistDialog.cs:74]: okay :)

**Pull request merged**